201-encrypt-running-linux-vm/azuredeploy.json

```diff
@@ -1,63 +1,99 @@
+        "diskFormatQuery": {
+            "defaultValue": "",
+            "metadata": {
+                "description": "the query string used to identify the disks to format and encrypt. This parameter only works when you choose the EncryptionOperation as EnableEncryptionFormat.[{\"scsi\":\"6:0:0:1\",\"name\":\"azuredisk\"},{\"scsi\":\"6:0:0:1\",\"name\":\"azuredisk2\"}], the scsi means the disk scsi number, the name means the mount point name"
+            },
+            "type": "string"
+        },
+        "encryptionOperation": {
+            "allowedValues": [ "EnableEncryption", "EnableEncryptionFormat" ],
+            "defaultValue": "EnableEncryption",
+            "metadata": {
+                "description": "EnableEncryption would encrypt the disks in place and EnableEncryptionFormat would format the disks directly"
+            },
+            "type": "string"
+        },
+        "keyEncryptionKeyURL": {
+            "defaultValue": "",
+            "metadata": {
+                "description": "URL of the KeyEncryptionKey used to encrypt the volume encryption key"
+            },
+            "type": "string"
+        },
+        "keyVaultResourceID": {
             "metadata": {
                 "description": "ResourceID of the KeyVault to place the volume encryption key"
-            }
+            },
+            "type": "string"
         },
-        "KeyVaultURL": {
-            "type": "string",
+        "keyVaultURL": {
             "metadata": {
                 "description": "URL of the KeyVault to place the volume encryption key"
-            }
+            },
+            "type": "string"
         },
-        "VolumeType": {
-            "type": "string",
-            "defaultValue": "Data",
+        "passphrase": {
+            "defaultValue": "",
             "metadata": {
-                "description": "Type of the volume to perform encryption operation. For Linux VMs allowed volume type is Data only"
-            }
+                "description": "The passphrase for the disks"
+            },
+            "type": "securestring"
         },
         "sequenceVersion": {
-            "type": "string",
             "defaultValue": "1",
             "metadata": {
                 "description": "sequence version of the bitlocker operation. Increment this everytime an operation is performed on the same VM"
-            }
+            },
+            "type": "string"
         },
-        "Passphrase": {
-            "type": "securestring",
-            "defaultValue": null,
+        "useExistingKek": {
+            "allowedValues": [
+                "nokek",
+                "kek"
+            ],
+            "defaultValue": "nokek",
             "metadata": {
-                "description": "the passphrase specified."
-            }
+                "description": "Select kek if the secret should be encrypted with a key encryption key"
+            },
+            "type": "string"
+        },
+        "vmName": {
```

Comment:
If this is the name of the existing running VM, the parameter should have running in it.